src/views/device/service-details/components/serviceModal.vue

```diff
@@ -104,12 +97,9 @@ defineExpose({ openModal });
           <n-form-item label="请输入key" path="service_identifier">
             <n-input v-model:value="form.service_identifier" placeholder="请输入服务标识符" />
           </n-form-item>
-          <n-form-item label="类别" path="service_type">
-            <n-space vertical class="selectType" placeholder="请选择服务类别">
-              <n-select v-model:value="form.service_type" :options="options" />
-            </n-space>
-          </n-form-item>
         </n-form>
+        <FormInput v-model:protocol-config="protocol_config" :form-elements="formElements"></FormInput>
```

Comment:
Change the code to below:

<FormInput v-model:protocol-config="config_data" :form-elements="formElements"></FormInput>